plugins/processors/t128_filter/README.md

```diff
@@ -9,12 +9,23 @@ The `t128_filter` filters out metrics passing through it. This processor is usef
 ```toml
 [[processors.t128_filter]]
   ## The conditions that must be met to pass a metric through. This is similar
-  ## behavior to a tagpass, but multiple tags are ANDed
+  ## behavior to a tagpass, but the multiple tags are ANDed
   [[processors.t128_filter.condition]]
+    ## Mode dictates how to match the condition's tag values
+    ## Valid values are:
+    ##  * "exact": exact string comparison
+    ##  * "glob": go flavored glob comparison (see https://github.com/gobwas/glob)
```

Comment:
I know this links to the github page but I think having some examples of globs and regex would be useful.